third_party/googletest/src/googletest/src/gtest.cc

```diff
@@ -182,6 +188,8 @@ const char kStackTraceMarker[] = "\nStack trace:\n";
 // is specified on the command line.
 bool g_help_flag = false;
 
+
+#if !GTEST_OS_STARBOARD
 // Utility function to Open File for Writing
 static FILE* OpenFileForWriting(const std::string& output_file) {
```

Comment:
What is this used for, and would it be useful to provide a Starboard equivalent for this ? What functionality depends on this? No need to do right away, but if there value in here, i'd add a TODO(b/1234) for further improvements